docs/dev/cli/cli-commands.md

```diff
@@ -15,10 +14,10 @@ Steps to build the **TOSCAna Command Line Interface** from sources:
 3. Build the project with:
 
     `
-    mvn package
+    mvn install
```

Comment:
Did you try this recently? I tried it and it does not work. 
```
mvn install
```

log:
```
Failed tests: 
  HelpTest.helpTest:43 
Expected: is <true>
     but: was <false>

Tests run: 5, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 4.419 s
[INFO] Finished at: 2018-03-30T19:41:56+02:00
[INFO] Final Memory: 18M/224M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.18.1:test (default-test) on project cli: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/heiko/TOSCAna/cli/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

I suggest we skip test suite execution (or someone fixes #622). That works as follows:
```
mvn install -Dmaven.test.skip=true
```

If someone can explain why `mvn install -dskipTests` does not work, I will ship a 🍪 to that sir.

Reply:
added this line instead of mvn install